pyuvdata/uvdata/uvdata.py

```diff
@@ -34,6 +34,7 @@
 logger = logging.getLogger(__name__)
 
 allowed_cat_types = ["sidereal", "ephem", "unprojected", "driftscan"]
+EMPTY_INDX = np.array([], dtype=np.int64)
```

Comment:
Doesn't look like this is used anywhere.

Reply:
Removed.